MAP/src/App.js

```diff
@@ -49,6 +48,7 @@ const getDaysDiff = (date1, date2) => {
   );
 };
 const { PLAYING, PAUSED } = playerStates;
+const  coords = { lng: 40.7, lat: 25, zoom: 1.06 }; //default coordinates
```

Comment:
this should be the default for `projectlockdown.earth`, but should also be added to these so that we can leverage the other TLD default locations easier later